.github/workflows/jit.yml

```diff
@@ -39,129 +48,122 @@ jobs:
+          ./PCbuild/build.bat --experimental-jit ${{ matrix.debug && '-d' || '' }} -p ${{ matrix.arch }}
 
+  jit-mac:
+    name: macOS (${{ matrix.os }}, ${{ matrix.debug && 'Debug' || 'Release' }})
+    needs: interpreter
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 90
+    strategy:
+      fail-fast: false
+      matrix:
+        # macos-14 are M1, macos-13 are default GHA Intel.
+        os:
+        - macos-14
+        - macos-13
+        debug:
+          - true
+          - false
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
         # The `find` line is required as a result of https://github.com/actions/runner-images/issues/9966.
         # This is a bug in the macOS runner image where the pre-installed Python is installed in the same
-        # directory as the Homebrew Python, which causes the build to fail for macos-13. This line removes 
+        # directory as the Homebrew Python, which causes the build to fail for macos-13. This line removes
         # the symlink to the pre-installed Python so that the Homebrew Python is used instead.
-      - name: Native macOS
-        if: runner.os == 'macOS'
+      - name: ${{ runner.os }} (${{ runner.arch }})
         run: |
           brew update
           find /usr/local/bin -lname '*/Library/Frameworks/Python.framework/*' -delete
-          brew install llvm@${{ matrix.llvm }}
+          brew install llvm@${{ env.LLVM }}
           export SDKROOT="$(xcrun --show-sdk-path)"
           ./configure --enable-experimental-jit ${{ matrix.debug && '--with-pydebug' || '' }}
           make all --jobs 4
           ./python.exe -m test --multiprocess 0 --timeout 4500 --verbose2 --verbose3
 
-      - name: Native Linux
-        if: runner.os == 'Linux' && matrix.architecture == 'x86_64'
+  jit-ubuntu:
+    name: Ubuntu (${{ matrix.os }}, ${{ matrix.debug && 'Debug' || 'Release' }})
+    needs: interpreter
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 90
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+        - ubuntu-24.04
+        - ubuntu-24.04-aarch64
+        debug:
+          - true
+          - false
+        is-fork:  # only used for the exclusion trick
+        - ${{ github.repository_owner != 'python' }}
+        exclude:
+        - os: ubuntu-24.04-aarch64
+          is-fork: true
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+
+      - name: ${{ runner.os }} (${{ runner.arch }})
         run: |
-          sudo bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)" ./llvm.sh ${{ matrix.llvm }}
-          export PATH="$(llvm-config-${{ matrix.llvm }} --bindir):$PATH"
+          sudo bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)" ./llvm.sh ${{ env.LLVM }}
+          export PATH="$(llvm-config-${{ env.llvm }} --bindir):$PATH"
```

Comment:
Nit, but can we use all lowercase for LLVM? The line above and the env var use upper case.

Reply:
Sure, I'll put everything lowercase.